pkg/util/container.go

```diff
@@ -48,6 +48,14 @@ func GetContainerCgroupPathWithKubeByID(podParentDir string, containerID string)
 	), nil
 }
 
+func GetContainerCgroupCPUProcsPath(podParentDir string, c *corev1.ContainerStatus) (string, error) {
```

Comment:
can you add an input/output example in comment? thanks.

Reply:
done